src/Tinify/Client.cs

```diff
@@ -16,7 +16,7 @@ public class Client : IDisposable
         public static readonly Uri ApiEndpoint = new Uri("https://api.tinify.com");
 
         public static readonly ushort RetryCount = 1;
-        public static readonly ushort RetryDelay = 500;
+        public static ushort RetryDelay { get; internal set; }= 500;

```

Comment:
This change was needed in order to support the testing code that was using reflection to set this value. Due to a change in how reflection works, the testing helper code would throw an exception due to trying to set an initonly value.